outboundSender.go

```diff
@@ -480,7 +480,7 @@ Loop:
 
 			if now.Before(dropUntil) {
 				obs.droppedCutoffCounter.Add(1.0)
-				break Loop
+				continue
```

Comment:
This works, but do we still need the `Loop` label at all?

Quick thought:  Seems like this code never exits now, which isn't much of an issue if we never intend to quiesce/restart it.